beoordelingstool.py

```diff
@@ -241,55 +234,25 @@ def unload(self):
     def run(self):
         """Run method that loads and starts the plugin"""
 
-        manholes_layerList = QgsMapLayerRegistry.instance().mapLayersByName("manholes")
-        pipes_layerList = QgsMapLayerRegistry.instance().mapLayersByName("pipes")
-        measuring_stations_layerList = QgsMapLayerRegistry.instance().mapLayersByName("measuring_points")
+        manholes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_MANHOLES)
+        pipes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_PIPES)
+        measuring_points_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_MEASURING_POINTS)
         if not self.pluginIsActive:
 
             #print "** STARTING Beoordelingstool"
 
             # dockwidget may not exist if:
             #    first run of plugin
             #    removed on close (see self.onClosePlugin method)
-            # Check if the layers manholes, pipes and measuring_stations are active
-            if not manholes_layerList or not pipes_layerList or not measuring_stations_layerList:
+            # Check if the layers manholes, pipes and measuring_points are active
+            if not manholes_layerList or not pipes_layerList or not measuring_points_layerList:
                 iface.messageBar().pushMessage("Warning", "You don't have a manholes, pipes and measuring_points layer. \n Upload a json.", level=QgsMessageBar.WARNING, duration=0)
```

Comment:
Is de string `Warning` de gebruikelijke manier? Niet een of andere constante importeren à la `import logging.WARN`?

Reply:
"Warning" is het begin van het QGIS bericht, een titel als het ware.
Maar wel een goed idee om ook hier een constante van te maken!